ipify/client_rest.go

```diff
@@ -51,6 +52,18 @@ func (client *clientRest) GetIp() (string, error) {
 	return ipResponse.IP, nil
 }
 
+func (client *clientRest) GetOutboundIP() (string, error) {
+	conn, err := net.Dial("udp", "8.8.8.8:53")
```

Comment:
Is 53 DNS port? Can it be unavailable?

Reply:
It can, but it does not matter. It is datagram, it has no state. 8.8.8.8 can even be not reachable, but we will still be able to get outbound IP